documentation/datasets.md

```diff
@@ -0,0 +1,89 @@
+# Built-in Datasets
+
+## Object Detection
+### Paired Image-Label Dataset
```

Comment:
Don't be shy; call it YOLO format dataset.
even write explicitly, if you are working with Yolo v1-8, your data is probably organized in this way

Reply:
Changed in https://github.com/Deci-AI/data-gradients/pull/114/commits/6f40aaf70aa694e3f8a79dd37ece3155abcbc8ec